src/common/components/build-log/index.js

```diff
@@ -0,0 +1,18 @@
+import React, { PropTypes } from 'react';
+
+import styles from './build-log.css';
+
+const BuildLog= (props) => {
+
+  return (
+    <iframe className={styles.frame} src={props.logUrl}>
```

Comment:
Most basic component for displaying build log as an iframe.

It's obviously very limited, but the quickest thing we can have ATM.